### ISSUE TITLE
Optimization rasterization by Bounding Box

### ISSUE THREAD

**Issue by cleardusk** [daniilidis-group/neural_renderer#50]
From [this line](https://github.com/daniilidis-group/neural_renderer/blob/master/neural_renderer/cuda/rasterize_cuda_kernel.cu#L87), it seems that rasterization is applied for every pixel when a face comes. Can it be accelerated by using the bounding box of faces like [Optimizing: 2D Triangles Bounding Box](https://www.scratchapixel.com/lessons/3d-basic-rendering/rasterization-practical-implementation)?

**Comment by nkolot** [comment]
Yeah, such optimizations should be possible but I don't have the time to work on it. Feel free to implement it if you want and submit a PR.
